### PR TITLE
[TASK] Make it run

### PR DESCRIPTION
**Pull request by benjaminkott**

### CODE REVIEW

File: src/Controller/Admin/UserCrudController.php [hunk 1]
```diff
@@ -22,7 +30,8 @@ class UserCrudController extends AbstractCrudController
 {
     public function __construct(
         private UserPasswordHasherInterface $hasher,
-    ) {}
+    ) {
+    }
```
Comment (Crell) [inline src/Controller/Admin/UserCrudController.php:34]: Any coding guideline that requires this much pointless extra typing is wrong and should be fixed.  Yes, that includes PSR-12, which is wrong.

File: src/Entity/FeedEntry.php [hunk 2]
```diff
@@ -73,6 +81,9 @@ public function getDescription(): ?string
         $regex = "%<a[^<>]*href=\"?{$link}\"?.*{$title}.*</a>%isU";
         $description = preg_replace($regex, '', $this->description);
 
+        // Remove unnessesary whitespace from lines
+        $description = implode("\n", array_map(function ($line) { return trim($line); }, explode("\n", $description)));
```
Comment (Crell) [inline src/Entity/FeedEntry.php:85]:
```suggestion
        $description = implode("\n", array_map(trim(...), explode("\n", $description)));
```

File: src/Repository/FeedEntryRepository.php [hunk 3]
```diff
@@ -43,7 +51,7 @@ public function latestEntriesPaginator(int $offset): Paginator
      * Purge feed entries older than a certain timestamp.
      *
      * @param \DateTimeImmutable $threshold
-     *   The date older than which entries should be purged.
+     *                                      The date older than which entries should be purged
```
Comment (Crell) [inline src/Repository/FeedEntryRepository.php:54]: What the heck is going on here?  That's the CS fixer going rogue.

(Descriptions belong on the next line, with one indent stop.)

File: src/Repository/FeedEntryRepository.php [hunk 4]
```diff
@@ -59,7 +67,7 @@ public function deleteOlderThan(\DateTimeImmutable $threshold): void
      * Marks multiple entries as approved, so they show in feeds.
      *
      * @param string ...$ids
-     *   A list of the IDs (links) to approve.
+     *                       A list of the IDs (links) to approve.
```
Comment (Crell) [inline src/Repository/FeedEntryRepository.php:70]: Ibid.

File: tests/AdminTest.php [hunk 5]
```diff
@@ -45,7 +50,7 @@ public function anon_user_gets_login_form_on_admin(): void
     /**
      * @test
      */
-    public function feed_index_loads(): void
+    public function feedIndexLoads(): void
```
Comment (Crell) [inline tests/AdminTest.php:53]: Lately I've found the snake-case style for test methods to be easier to read.  I've seen lots of recommendations in every direction on this; I'd rather not mess with this along with everything else.

File: composer.json [hunk 6]
```diff
@@ -41,12 +41,16 @@
...
-        "symfony/phpunit-bridge": "^5.4 || ^6.0",
         "doctrine/doctrine-fixtures-bundle": "^3.4",
-        "overtrue/phplint": "^4.1"
+        "friendsofphp/php-cs-fixer": "^3.6",
+        "overtrue/phplint": "^4.1",
+        "phpspec/prophecy-phpunit": "^2.0",
+        "phpstan/phpstan": "^1.0",
+        "phpunit/phpunit": "^9.5",
+        "symfony/browser-kit": "^5.4 || ^6.0",
+        "symfony/css-selector": "^5.4 || ^6.0",
```
Comment (Crell) [inline composer.json:51]: Note: Added these 2 packages.

File: src/Controller/Admin/UserCrudController.php (hunk 1 above)
Comment (benjaminkott) [reply src/Controller/Admin/UserCrudController.php:34]: What ever you like, just adapt it and make it auto fixable

File: src/MessageHandler/PurgeOldEntriesHandler.php [hunk 7]
```diff
@@ -13,7 +21,7 @@
 
 final class PurgeOldEntriesHandler implements MessageHandlerInterface
 {
-    protected FeedEntryRepository $entryRepo;
+    private FeedEntryRepository $entryRepo;
```
Comment (benjaminkott) [inline src/MessageHandler/PurgeOldEntriesHandler.php:24]: please check

File: src/MessageHandler/UpdateFeedHandler.php [hunk 8]
```diff
@@ -78,7 +88,7 @@ public function __invoke(UpdateFeed $message): void
         });
     }
 
-    protected function updateFeedEntry(EntryInterface $item, EntityManagerInterface $em, Feed $feed): FeedEntry
+    private function updateFeedEntry(EntryInterface $item, EntityManagerInterface $em, Feed $feed): FeedEntry
```
Comment (benjaminkott) [inline src/MessageHandler/UpdateFeedHandler.php:91]: please check

File: src/MessageHandler/UpdateFeedHandler.php [hunk 9]
```diff
@@ -104,7 +114,7 @@ protected function updateFeedEntry(EntryInterface $item, EntityManagerInterface
         return $entry;
     }
 
-    protected function updateFeed(Feed $feed, FeedInterface $feedData): Feed
+    private function updateFeed(Feed $feed, FeedInterface $feedData): Feed
```
Comment (benjaminkott) [inline src/MessageHandler/UpdateFeedHandler.php:117]: please check

**Pull request merged**